lib/daru/category.rb

```diff
@@ -14,7 +14,7 @@ def initialize_category data, opts={}
       if opts[:categories]
         validate_categories(opts[:categories])
         add_extra_categories(opts[:categories] - categories)
-        self.order = opts[:categories]
+        set_order opts[:categories]
```

Comment:
I thought you had decided to stick with `order=`? Any reason for deviating from that?